ui/plugins/controller.js

```diff
@@ -48,7 +48,14 @@ treeherder.controller('PluginCtrl', [
         var selectJobPromise = null;
         var selectJobRetryPromise = null;
 
-        var selectJob = function(job_id) {
+        var selectJob = function(job_id, job_selection_type) {
+            // All selection in treeherder is generally 'active' via omission
+            // of the job_selection_type param but we set it here to not leave
+            // it undefined
+            if (!job_selection_type) {
+                // Currently unused
+                job_selection_type = 'active';
+            }
```

Comment:
@camd not sure if we really need to include this or just let it exist as `undefined` for callers which don't pass the 2nd parameter (which is most of them in treeherder at the moment). I recalled from our discussion on Friday so I added it in during this WIP.

I spent a bunch of time word smithing a useful param value - and ultimately came up with the concept of 'active' and 'passive' selection types. Where 'active' generally implies what the user did, and 'passive' is what treeherder decided to do for the user.